src/app/core/services/app-config/feature-flag.service.ts

```diff
@@ -0,0 +1,15 @@
+import { Inject, Injectable } from '@angular/core';
+import { AppConfigService } from '@services/app-config/app-config.service';
+
+@Injectable()
+export class FeatureFlagService {
+  constructor(@Inject(AppConfigService) private appConfigService: AppConfigService) {}
```

Comment:
Small observation

`private appConfigService = inject(AppConfigService);` could be used here to make this less verbose.

The later versions of angular are moving away from constructor injection in favour of inject() function. 
Don't feel you have to change it as either way is fine :)

Reply:
done